SimulationAgent/Agent.cs

```diff
@@ -205,9 +221,13 @@ private void CheckForNewSimulation(Simulation newSimulation)
                 this.simulation = newSimulation;
                 if (this.simulation.ShouldBeRunning())
                 {
-                    this.log.Debug("------ Starting new simulation ------", () => new { this.simulation });
+                    var simulationStatus = "------ Starting new simulation ------";
```

Comment:
this message is styled with "------" to be visible in the logs, but I doubt that's what we want to see in the backend